### PR TITLE
ping return FALSE if error occurred and the unit test

### PR DESCRIPTION
**Pull request by wush978**
- ping return FALSE if error occurred
- unit test of ping
ropensci/ckanr#1

### CODE REVIEW

File: R/ping.R [hunk 1]
```diff
@@ -12,6 +12,8 @@
 #' }
 ping <- function(url = get_ckanr_url(), as="logical", ...)
 {
-  res <- ckan_POST(url, 'site_read', ...)
-  switch(as, json = res, logical = jsd(res))
+  retval <- tryCatch({
```
Comment (sckott) [inline R/ping.R]: @wush978 nothing is returned from this function though, you assign to `retval` , so the value isn't returned

Comment (wush978) [inline R/ping.R]: Oh, I accidentally do it right.

In R, `a <- 1` will return `1`. But you are right, I forgot to put `retval` at the end.

Comment (sckott) [inline R/ping.R]: can you fix - or I can merge and fix after

Comment (wush978) [inline R/ping.R]: Sure. 75994da fixed this

**Pull request merged**